docs/source/getting_started/teams.rst

```diff
@@ -0,0 +1,132 @@
+========
+Teams
+========
+
+Teams overview
+
+.. warning::
+    Teams are not available on all Citrine Platform deployments.
+
+    Please contact support for more information.
+
+Basic Team Use
+-----------------
+
+Most commonly, the first thing you will want to do after connecting to the Citrine Platform
+is to find a certain Team. Assume that you have created a :class:`~citrine.citrine.Citrine`
+client object named ``citrine``. To list all of the Teams that you are a member of, use the
+``list`` command.
+
+.. code-block:: python
+
+    citrine.teams.list()
+
+To retrieve a Team that you are a member of, either find the Team in the list:
```

Comment:
Does this imply that you can only retrieve a team of which you are a member? If so, I think we should say that explicitly. Otherwise, we should drop "that you are a member of".

Reply:
yes, you can only list teams you are a member of (so that there's no info leak with team names I guess?) There's accounts support for listing teams as an admin user, but that isn't propagated to the python sdk